cumulusci/tasks/salesforce/UpdateAdminProfile.py

```diff
@@ -181,3 +184,8 @@ def _deploy_metadata(self):
         self.logger.info("Deploying updated Admin.profile from {}".format(self.tempdir))
         api = self._get_api(path=self.tempdir)
         return api()
+
+    def freeze(self, step):
+        if self._default_package_xml:
+            del self.options["package_xml"]
```

Comment:
`freeze` should not have the side effect of mutating the task instance's options. Which makes it harder to just call super() here. Why not just leave self.options['package_xml'] unset in _init_options, and make _init_task like this:

    path = self.options.get('package_xml') or os.path.join(CUMULUSCI_PATH, "cumulusci", "files", "admin_profile.xml")
    with open(path, "r") as f:
        self._package_xml_content = f.read()

Actually you can put that at the end of _init_options. That way it errors quickly if the filename is wrong, but also avoids the need to track _default_package_xml and use a custom freeze method.

Reply:
Done in latest commit